xbmc/guilib/FFmpegImage.cpp

```diff
@@ -309,9 +334,175 @@ bool CFFmpegImage::CreateThumbnailFromSurface(unsigned char* bufferin, unsigned
                                              unsigned char* &bufferout,
                                              unsigned int &bufferoutSize)
 {
-  return false;
+  // It seems XB_FMT_A8R8G8B8 mean RGBA and not ARGB
```

Comment:
I think it's becuase of https://ffmpeg.org/doxygen/2.8/pixfmt_8h.html#a9a8e335cf3be472042bc9f0cf80cd4c5